schematic/utils/google_api_utils.py

```diff
@@ -24,30 +24,6 @@
     "https://www.googleapis.com/auth/drive",
 ]
 
-
```

Comment:
I assume this is removed bc its no longer part of the auth process. If so I think it would be nice to call out this change explicitly within the description for this PR.

Reply:
That's correct it wasn't being used anymore. I've linked to the refactor document in the first comment. It has alot of what should be in the description.